lib/tasks/wikirate.rake

```diff
@@ -17,134 +17,82 @@ namespace :wikirate do
-          card.destroy
-        end
-        Rake::Task['wagn:migrate'].invoke
-
-        # select 5 companies and topics and metrics
-        companies = [Card["Apple Inc."],Card["Amazon.com, Inc."],Card["Samsung"],Card["Siemens AG"],Card["Sony Corporation"]]
-        topics = [Card["Natural Resource Use"],Card["Community"],Card["Human Rights"],Card["Climate Change"],Card["Animal Welfare"]]
-        metrics = [Card['Sebastian Jekutsch+CSR Report Available'],Card['Good Company Index+Good Company Grade'],Card['BSR+BSR Member']]
-        company_ids = ""
-        topic_ids = ""
-        metric_ids = ""
-        company_names = Array.new
-        topic_names = Array.new
-
-        card_to_be_kept = Array.new
-
-        companies.each do |company|
-          company_ids += "#{company.id},"
-          company_names.push company.name
-        end
-
-        topics.each do |topic|
-          topic_ids += "#{topic.id},"
-          topic_names.push topic.name
-        end
-
-
-        # metrics.each do |metric|
-        #   metric_ids += "#{metric.id},"
-        #   card_to_be_kept << metric.id
-        #   card_to_be_kept += Card.search :left=>metric.name, :return=>:id
-        # end
-
-
-
-        company_related_article = Card.search :type=>"Analysis",:left=>{:name=> company_names.unshift("in")}
-        topic_related_article = Card.search :type=>"Analysis",:right=>{:name=>topic_names.unshift("in")}
-
-        (companies + topics ).each do |c|
-          search_args = {:type=>["in","claim","source"]}
-          query = Card.tag_filter_query(c.name, search_args,['company','topic'])
-          cards = Card.search query
-          cards.each do |card|
-            card_to_be_kept.push card.id
+        
+        puts "getting production export".green
+        # we can let semaphore make the latest test db on the fly
+        export_location = case args[:location]
+                          when "dev"
+                            "dev.wikirate.org"
+                          when "production"
+                            "wikirate.org"
+                          else
+                            "127.0.0.1:3000"
+                          end          
+        export = open("http://#{export_location}/production_export.json?export=true",:read_timeout => 50000).read
+        puts "Done".green
+        cards = JSON.parse(export)
+        Card::Auth.as_bot
+        cards["card"]["value"].each do |c|
+          card_name = c["name"]
+          begin
+            if card = Card.fetch(card_name)      
+              puts "updating card #{c} #{card.update_attributes!(c)}".light_blue
+            else
+              puts "creating card #{c} #{Card.create!(c)}".yellow
+            end
+          rescue => e
+            puts "Error in #{c} #{e}".red
           end
         end
+        FileUtils.rm_rf(Dir.glob('tmp/*'))
+        seed_test_db = "RAILS_ENV=test rake wikirate:test:add_wikirate_test_data"
+        puts seed_test_db.green
+        system seed_test_db
 
```

Comment:
eg, the latest wagn master includes significant migrations.  

I think I'm going to go ahead and merge this without the latest wagn so we can get the special pointer ready on dev at least.  But we need to think about how to handle these migration situations.